src/MovieCard/MovieCard.test.js

```diff
@@ -0,0 +1,137 @@
+    const mockPosterPath = "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg";
+    const mockTitle = "Rogue";
+    const mockAverageRating = 5;
+    const mockID = 718444
+    const mockUser = { email: "charlie@turing.io", id: 81, name: "Charlie"};
+    const mockRetrieveAllRatings = jest.fn();
+
+    render(
+      <MemoryRouter>
+        <MovieCard
+          posterPath={mockPosterPath}
+          title={mockTitle}
+          averageRating={mockAverageRating}
+          id={mockID}
+          user={mockUser}
+          retrieveAllRatings={mockRetrieveAllRatings}
+        />
+      </MemoryRouter>
+    )
+    userEvent.click(screen.getByText('Submit'))
+    expect(screen.getByText("Choose a number between 1 and 10")).toBeInTheDocument()
+
+  })
+
+  it('Should display instructions if the input is an invalid number', () => {
+    const mockPosterPath = "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg";
+    const mockTitle = "Rogue";
+    const mockAverageRating = 5;
+    const mockID = 718444
+    const mockUser = { email: "charlie@turing.io", id: 81, name: "Charlie"};
+    const mockRetrieveAllRatings = jest.fn();
+
+    render(
+      <MemoryRouter>
+        <MovieCard
+          posterPath={mockPosterPath}
+          title={mockTitle}
+          averageRating={mockAverageRating}
+          id={mockID}
+          user={mockUser}
+          retrieveAllRatings={mockRetrieveAllRatings}
+        />
+      </MemoryRouter>
+    )
+
+    fireEvent.change(screen.getByRole('spinbutton'), {target: {value: 26}})
+    expect(screen.getByRole('spinbutton')).toHaveValue(26)
+    userEvent.click(screen.getByText('Submit'))
+    expect(screen.getByText("Choose a number between 1 and 10")).toBeInTheDocument()
+  })
+
+  it('Should allow a user to submit a rating', async () => {
+    const mockPosterPath = "https://image.tmdb.org/t/p/original//uOw5JD8IlD546feZ6oxbIjvN66P.jpg";
+    const mockTitle = "Rogue";
+    const mockAverageRating = 5;
+    const mockUserRating = undefined;
+    const mockID = 718444;
+    const mockUser = { email: "charlie@turing.io", id: 81, name: "Charlie"};
+
+    const mockRetrieveAllRatings = jest.fn();
+    const mockSubmitUserRating = jest.fn();
+    submitUserRating.mockResolvedValueOnce({rating: {user_id: 81, movie_id: 718444, rating: 7}})
+    // retrieveAllRatings.mockResolvedValueOnce([{ratings: {id: 1, user_id: 81, movie_id: 718444, rating: 7}}])
+    render(
+      <MemoryRouter>
+        <MovieCard
+          // submitRating={mockSubmitRating}
+          posterPath={mockPosterPath}
+          title={mockTitle}
+          averageRating={mockAverageRating}
+          id={mockID}
+          user={mockUser}
+          retrieveAllRatings={mockRetrieveAllRatings}
+          userRating={mockUserRating}
+        />
+      </MemoryRouter>
+    )
+
+
+fireEvent.change(screen.getByRole('spinbutton'), {target: {value: 7}})
```

Comment:
I'm wondering if there is a reason for using `fireEvent` rather than `userEvent` here on 122.  Is there a `change()` method for `userEvent` ?